src/net-questdb-client/Buffers/BufferV1.cs

```diff
@@ -317,7 +317,7 @@ public IBuffer Column(ReadOnlySpan<char> name, DateTime timestamp)
         }
 
         var epoch = timestamp.Ticks - EpochTicks;
-        Column(name).Put(epoch / 10).PutAscii('t');
+        Column(name).Put(epoch * 100).PutAscii('n');
```

Comment:
nit: to make consistent with At() method
```suggestion
        Column(name).Put(epoch).Put('0').Put('0').Put('n');
```